.github/workflows/releaseTrigger.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Release
+
+on:
+  create:
+    tags:
+      - 'releaseTrigger'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: releasing
+    steps:
+      - name: Wait for build to succeed
+        uses: jitterbit/await-check-suites@v1
+        with:
+          appSlugFilter: 'github-actions'
+          timeoutSeconds: '300'
```

Comment:
This means we wait for the build in `actions.yml` to complete successfully? Is that job uniquely identified, i.e. what happens if we ever add a third `yml` for whatever reason? Could this step here then be completed by that other job workflow?

Reply:
> This means we wait for the build in actions.yml to complete successfully?

yes

> Is that job uniquely identified, i.e. what happens if we ever add a third yml for whatever reason?

No currently we wait for all test suites which are running on the same git SHA to be finished, not one particular, but all checks from github actions

> Could this step here then be completed by that other job workflow?

Not completed but it will be a necessary step to wait for

Reply:
That's good to know (can you add a short comment explaining that?). I wondered whether we should try to limit this to the actual checks that are currently configured, but I think waiting for all to be successful (they need to be successful, right?) is safer. This way, when we add a new job, the worst that can happen is the release not getting triggered.

Reply:
added comment